pkg/app/client.go

```diff
@@ -0,0 +1,12 @@
+package app
+
+import "os/exec"
+
+func GetCOEVersion() ([]byte, error) {
+	return runCommand("oc", "version")
+}
+
+func runCommand(cmd string, args ...string) ([]byte, error) {
+	output, err := exec.Command(cmd, args...).CombinedOutput()
+	return output, err
+}
```

Comment:
I think a utils pkg at the top level is a good idea.  I created https://github.com/fusor/ansible-service-broker/issues/62 to track that.  I think we should have a separate patch to fix that.